templates/AureliaSpa/ClientApp/app/components/fetchdata/fetchdata.ts

```diff
@@ -0,0 +1,27 @@
+/// <reference path="../../../../node_modules/aurelia-fetch-client/doc/whatwg-fetch.d.ts" />
+/// <reference path="../../../../node_modules/aurelia-fetch-client/doc/url.d.ts" />
```

Comment:
Yes, in general the change in type references for TS 2.0 was something I only realized this Saturday as I was cleaning up some things.  Will look at.